src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -102,7 +102,7 @@ public function setChanges(array $changes)
     /**
      * Sets a factory.
      *
-     * @param string|array|Reference $factory A PHP function, reference or an array containing a class/Reference and a method to call
+     * @param string|array|Reference|null $factory A PHP function, reference or an array containing a class/Reference and a method to call
```

Comment:
This is also a minor. 

The `SessionPass` is passing "null" as parameter to this method to reset the factory.